editors/sc-ide/core/doc_manager.cpp

```diff
@@ -773,7 +773,7 @@ void DocumentManager::handleNewDocScRequest( const QString & data )
             Q_EMIT( opened(document, 0, 0) );
         }
     } catch (std::exception const& e) {
-        qWarning() << "DocumentManager::" << __func__ << ": could not handle request:" << e.what();
+        qWarning() << "DocumentManager::" << __FUNCTION__ << ": could not handle request:" << e.what();
```

Comment:
tbth i should have just replaced this with a macro and pulled it out into a separate header. but i needed to do something about it to get it to compile with VS, and it was the quickest solution, and i'd rather tackle this cleanliness issue in a separate PR at this point.